frontend/src/model/beta/overview/FormEditPage.tsx

```diff
@@ -118,9 +120,11 @@ export default function FormEditPage({ model }: FormEditPageProps) {
               <Button variant='outlined' onClick={onCancel}>
                 Cancel
               </Button>
-              <Button variant='contained' onClick={onSubmit}>
-                Save
-              </Button>
+              {!loading && (
```

Comment:
Why are you conditionally rendering your `LoadingButton`? This can be removed as we always want to render the `LoadingButton`, regardless of whether or not we're in a loading state.